### PR TITLE
Setup page loader added

### PR DESCRIPTION
**Pull request by hardikhdholariya**
- Added threadpool to handle the multiple requests
- Added spinner to wait until disable/enable the product
- removed duplicate key

### CODE REVIEW

File: product_setup_page/src/components/ProductSetup.jsx [hunk 1]
```diff
@@ -87,6 +93,7 @@ export default function ProductSetup(props) {
       ))}
       <Button label="Save" appearance="primary" onClick={saveMacros} updateMacroDefinition={updateMacroDefinition} />
       {response && <pre>{response}</pre>}
+      {isLoading ? <div id="spinner"></div>: null}
```
Comment (VatsalJagani) [inline product_setup_page/src/components/ProductSetup.jsx:96]: Please send me screenshot of loader.

**Review by VatsalJagani — APPROVED** [review]

File: cyences_app_for_splunk/bin/rh_cyences_product_conf.py [hunk 2]
```diff
@@ -80,22 +81,38 @@ def configure_saved_searches(self, enabled_products, disabled_products):
...
-                self.conf_manager.update_savedsearch(name, {"disabled": new_disabled})
-                changed_savedsearches.append(name + " => " + ("Disabled" if new_disabled else "Enabled"))
 
-        return '\n'.join(changed_savedsearches)
+            if current_disabled != new_disabled:
+                changed_savedsearches.append([name, new_disabled])
+                messages_to_display.append(name + " => " + ("Disabled" if new_disabled else "Enabled"))
+
+        if len(changed_savedsearches) > 0:
+            with concurrent.futures.ThreadPoolExecutor(max_workers=min(len(changed_savedsearches), 15)) as executor:
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/bin/rh_cyences_product_conf.py:102]: Isn't 15 is high number? Shall we reduce it?

File: product_setup_page/src/components/ProductSetup.jsx (hunk 1 above)
Comment (hardikhdholariya) [reply product_setup_page/src/components/ProductSetup.jsx:96]: image

File: cyences_app_for_splunk/bin/rh_cyences_product_conf.py (hunk 2 above)
Comment (hardikhdholariya) [reply cyences_app_for_splunk/bin/rh_cyences_product_conf.py:102]: It always gives error with 20 threads. I tested with 15 and it's working fine.

**Pull request merged**